src/nupic/types/Serializable.hpp

```diff
@@ -37,7 +36,15 @@
 #include <nupic/os/Path.hpp>
 #include <nupic/os/ImportFilesystem.hpp>
 
-#define SERIALIZABLE_VERSION 1
+#define CEREAL_SAVE_FUNCTION_NAME save_ar
+#define CEREAL_LOAD_FUNCTION_NAME load_ar
```

Comment:
do we need the chain Serializable::save -> cereal::save_ar, or should serializable already dictate the cereal methods?

Reply:
My opinion is that the type of output (type of Archive object) depends on how you use in the operator>> and operator<< for text versions and saveToFile() and loadFromFile() in Serialization.